views/cabin/Event Guest invite shower.tw

```diff
@@ -21,14 +26,20 @@
 	<<link 'Go to the shower'>>
 		<<set $oldTmpGirlLocation = $tmpGirl.location>>
 		<<set $tmpGirl.location = 'shower'>>
-		<<goto 'Girl fuck'>>
+		<<if !$tmpGirl.gender>>	
+			<<goto 'Girl fuck'>>
+		<<else>>				
+			<<goto 'Guy fuck'>>
+		<</if>>
 	<</link>>
 <</if>>
 <<link 'Back'>>
   	<<if $tmpGirlViewBack>>
 		<<goto $tmpGirlViewBack>>
-	<<else>>
+	<<elseif !$tmpGirl.gender>>
 		<<goto [[Girl view]]>>
```

Comment:
This won't work here as Girl fuck/Guy fuck moves to passage that is designed for slaves.
$tmpGirlViewBack should probably work for guys too. so you just need to set correct $tmpGirlViewBack